share/spice/sales_tax_holiday/sales_tax_holiday.js

```diff
@@ -0,0 +1,88 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_sales_tax_holiday = function(api_result){
+        // Error checking
+        var errorMessage = "UNKNOWN ERROR";
+        var isError = false;
+        var isMultipleResults = false;
+        if (!api_result 
+         || (typeof api_result.header === 'undefined')
+         || api_result.header.status !== "SUCCESS" 
+         || !api_result.taxHolidays
+         || (typeof api_result.taxHolidays === 'undefined')
+         || api_result.taxHolidays.length < 1) {
+            if(api_result.header.errorMessage 
+            && api_result.header.errorCode 
+            && api_result.header.errorCode === 'NOT_FOUND') {
+                isError = true;
+                errorMessage = api_result.header.errorMessage;
+            } else {         
+              return Spice.failed('sales_tax_holiday');
+            }
+        }
+
+        if (api_result.taxHolidays.length > 1) {
+            isMultipleResults = true;
+        }
+        // Display
+      DDG.require("moment.js", function(){
+        Spice.add({
+            id: "sales_tax_holiday",
+            name: "Answer",
+            data: isError? {"errorMessage" : errorMessage} : api_result.taxHolidays,
+            meta: {
+                sourceName: "snapCX",
+                sourceIcon: false,
+                sourceIconUrl: "http://snapcx.io/favicon.ico",
+                sourceUrl: "https://snapcx.io"
```

Comment:
Can we link to a more informative page? This doesn't directly add any information about the answer.